### ISSUE TITLE
Compatibility with non-interactive docker containers

### ISSUE THREAD

**Issue by AGhost-7** [vxgmichel/aioconsole#102]
Certain programs can call your python program without a stdin. For example when you use `docker run` without the `-i` option you can get the following exception when calling a script using `create_standard_streams`:
```
Traceback (most recent call last):
  File "/usr/lib/python3.8/asyncio/selector_events.py", line 261, in _add_reader
    key = self._selector.get_key(fd)
  File "/usr/lib/python3.8/selectors.py", line 192, in get_key
    raise KeyError("{!r} is not registered".format(fileobj)) from None
KeyError: '0 is not registered'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/asyncio/events.py", line 81, in _run
    self._context.run(self._callback, *self._args)
  File "/usr/lib/python3.8/asyncio/selector_events.py", line 263, in _add_reader
    self._selector.register(fd, selectors.EVENT_READ,
  File "/usr/lib/python3.8/selectors.py", line 359, in register
    self._selector.register(key.fd, poller_events)
PermissionError: [Errno 1] Operation not permitted
```

**Comment by vxgmichel** [comment]
@AGhost-7 Thanks for the report !

Interesting, but how would you expect `create_standard_streams` in this case? Do you still need async access to stdout and stderr even though stdin is not available?

**Comment by AGhost-7** [comment]
In that case I would probably expect to get `None` for stdout and stderr. Its pretty common that applications will display things to the terminal differently based on whether or not a sessions is interactive, so it is useful to expose that information. Alternatively you could provide a noop shim object, but I think the better option is the former.

**Comment by DCsunset** [comment]
I've encountered the same issue. This exception could also occur when running the program in the background in a script.

Is there any idea how to detect whether stdin is readable? If so, it can actually be fixed outside this library by checking it first before `create_standard_streams`.

**Comment by DCsunset** [comment]
Yeah I think it's a good idea! I've also tested the code in your PR and it works without problem

**Comment by vxgmichel** [comment]
Fixed in #106, available in [v0.6.1](https://github.com/vxgmichel/aioconsole/releases/tag/v0.6.1) :tada:
